src/HomeWork_2/middlewares/log-error.ts

```diff
@@ -0,0 +1,8 @@
+import { Response, NextFunction } from 'express';
+import { errorHandler } from '../services/error-logger';
+
+export const logError = (err: Error, _: unknown, res: Response, next: NextFunction) => {
+  res.status(500).end('internal Server Error');
```

Comment:
FYI, you can use `.sendStatus(500)`, so that you don't have to specify the default message manually